copilot/fastapi/app/tvp.py

```
@@ -0,0 +1,392 @@
+                        with open(values_file, 'r') as f:
+                            try:
+                                values = yaml.safe_load(f)
+                                applications.append({
+                                    "app_name": app_dir.name,
+                                    "namespace": namespace_dir.name,
+                                    "image": values.get("image", "unknown"),
+                                    "tag": values.get("tag", "unknown"),
+                                })
+                            except yaml.YAMLError as yaml_err:
+                                # Log specific YAML parsing error
+                                logger.error(f"Error parsing YAML in {values_file}: {yaml_err}")
```

Ruff (0.8.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)